src/errormap.jl

```diff
@@ -1,6 +1,6 @@
-	# Error maps (maps err to (NAME, Description))
+        # Error maps (maps err to (NAME, Description))
 
-const error_codes = {
+const error_codes = Dict (
```

Comment:
That's basically it. For Dicts, I think you need `@compat` there but I can't tell you why.